common/models/user.js

```diff
@@ -556,25 +560,19 @@ module.exports = function(User) {
     // TODO - support more verification types
     function sendEmail(user) {
       verifyOptions.verifyHref += '&token=' + user.verificationToken;
-
       verifyOptions.verificationToken = user.verificationToken;
 
       verifyOptions.text = verifyOptions.text || g.f('Please verify your email by opening ' +
         'this link in a web browser:\n\t%s', verifyOptions.verifyHref);
 
       verifyOptions.text = verifyOptions.text.replace(/\{href\}/g, verifyOptions.verifyHref);
 
-      verifyOptions.to = verifyOptions.to || user.email;
-
-      verifyOptions.subject = verifyOptions.subject || g.f('Thanks for Registering');
-
-      verifyOptions.headers = verifyOptions.headers || {};
-
```

Comment:
these 3 assignments should be done while still sync so we can call assertVerifyOptions() before going async
(here we are already async as we saved the user with new verification token)